### PR TITLE
Feat dockerfile and refacto

### PR DESCRIPTION
**Pull request by Creadeyh**
Added Dockerfile and dockerignore, app is running in docker and is compatible with Python >= 3.8.
Cleaned some code and updated some names for better clarity.
Fixed Comment and Project creation in SQLiteRepository.
Fixed routes unit tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,40 +4,34 @@ This API is the backbone of the Survey tool. It collects user feedbacks from the
...
-You can retrieve all comments from the database using the GET /comments endpoint. The API returns a list of comment objects.
-
-### Comment Validation
-The API validates that the rating in a comment is an integer between 1 and 5. If the rating is invalid, the API returns an error message.
-
-### Project Creation
-The API automatically creates a new project in the database when a comment is added for a new project. This ensures that there are no orphaned comments in the database.
 ## Usage
 
 You need to run this commnand to install all the dependencies :
```
Comment (MelisandeDes) [inline README.md:37]: command

File: README.md
Comment (MelisandeDes) [inline README.md:1]: add a section to explain the docker image, how to install it.

**Review by MelisandeDes — COMMENTED** [review]
add a section to explain the docker image, how to install it.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# syntax=docker/dockerfile:1
+   
+FROM python:3.8
+WORKDIR /survey-back-api
+COPY . .
+COPY .env.example .env
```
Comment (aurelien-alet) [inline Dockerfile:6]: Use .env file rather than .env.example

Comment (Creadeyh) [reply Dockerfile]: Done

File: routes/comments.py [hunk 3]
```diff
@@ -1,24 +1,28 @@
...
-    project_name = commentcookie.feature_url.split('/')[2]
-    new_comment = await comment_repo.create_comment(commentcookie, user_id, timestamp, project_name)
+async def create_comment(
+        comment_body: CommentPostBody,
+        user_id: Union[str, None] = Cookie(default=None),
+        timestamp: Union[str, None] = Cookie(default=None),
+        sqlite_repo: SQLiteRepository = Depends(Provide[Container.sqlite_repo])
+    ) -> Comment:
+    # TODO: get domain name from request instead
+    project_name = comment_body.feature_url.split('/')[2]
```
[the remark this answers is not in the document]
Comment (Creadeyh) [reply routes/comments.py]: Done. If project name / feature isn't found in config file, returns 404

**Pull request merged**